docs/docs/releases/0.1.6.md

```diff
@@ -0,0 +1,36 @@
+# Mathesar 0.1.6
+
+## Summary
+
+Mathesar 0.1.6 brings compatibility with Python versions 3.10 and 3.11, PostgreSQL 16 support, ease of use while working with longer texts on the record page, as well as Japanese translations within the application.
+
+_This page provides a comprehensive list of all changes in the release._
+
+## Upgrading to 0.1.6
+
+See our guide on [upgrading Mathesar to 0.1.6](../administration/upgrade/0.1.6.md).
+
+## Improvements
+
+- Mathesar is now compatible with Python versions: 3.10 and 3.11 along with 3.9 _[#3478](https://github.com/mathesar-foundation/mathesar/pull/3478)_
+- Added support for Japanese translations _[#3486](https://github.com/mathesar-foundation/mathesar/pull/3486)_, _[#3484](https://github.com/mathesar-foundation/mathesar/pull/3484)_, _[#3483](https://github.com/mathesar-foundation/mathesar/pull/3483)_, _[#3472](https://github.com/mathesar-foundation/mathesar/pull/3472)_
+- NodeJS is no longer a requirement for building Mathesar from source _[#3474](https://github.com/mathesar-foundation/mathesar/pull/3474)_
```

Comment:
This isn't actually what the linked PR is about is it? It's about removing Node from the prod Docker image.